App/src/main/java/cc/softwarefactory/lokki/android/NavigationDrawerFragment.java

```diff
@@ -85,8 +83,7 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
             }
         });
         String[] menuOptions = getResources().getStringArray(R.array.menuOptions);
-        //mDrawerListView.setAdapter(new ArrayAdapter<String>(getActionBar().getThemedContext(), android.R.layout.simple_list_item_1, android.R.id.text1, menuOptions));
-        mDrawerListView.setAdapter(new ArrayAdapter<String>(getActionBar().getThemedContext(), R.layout.drawer_list_item, menuOptions));
+        mDrawerListView.setAdapter(new ArrayAdapter<>(getActionBar().getThemedContext(), R.layout.drawer_list_item, menuOptions));
```

Comment:
Should be updated to use `getSupportActionBar()` instead of `getActionBar()` as in the [guidelines](http://android-developers.blogspot.fi/2014/10/appcompat-v21-material-design-for-pre.html) to ensure backwards compatibility.